deployment/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN cargo chef prepare --recipe-path recipe.json
 
 FROM chef as builder
 ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
+ENV CARGO_PROFILE_RELEASE_DEBUG=true
```

Comment:
Hmm, maybe we need to create two images. One with debug information and one without. Or keep with debug info until mainnet and split into two images before mainnat. But in this case, we need to create an issue not to forget to split it.

Reply:
If we do split debug info, the compilation time for docker images wont really change. It would only allow users to opt-out of a larger docker image, which I don't think is a huge deal?